src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/altar/AncientPedestal.java

```diff
@@ -123,7 +156,8 @@ private boolean testItem(@Nullable Entity n) {
 
     public void placeItem(@Nonnull Player p, @Nonnull Block b) {
         ItemStack hand = p.getInventory().getItemInMainHand();
-        ItemStack displayItem = new CustomItemStack(hand, ITEM_PREFIX + System.nanoTime());
+        String displayName = ITEM_PREFIX + System.nanoTime();
+        ItemStack displayItem = new CustomItemStack(hand, displayName);
```

Comment:
Any reason this is split up?

Reply:
Yeah if you scroll down it's used with the armorstand as well